docs/root/intro/version_history.rst

```diff
@@ -15,6 +15,7 @@ Bugfixes:
 
 - android: fix engine startup crash for when admin interface is enabled. (:issue:`#2520 <2520>`)
 - android: respect system security policy when determining whether clear text requests are allowed. (:issue:`#2528 <2528>`)
+- tls: fix a potential use-after-free with the platform cert validator. (:issue:`#2691 <2691>`)
```

Comment:
Not a big deal, but change log entries aren’t necessary for fixes to features added in that same version.

Reply:
Oh, I didn't realize that. Thanks! Reverted.